### PR TITLE
sphinx: add plantuml support

### PR DESCRIPTION
**Pull request by pbrezina**
```
.. uml::

    plantumlcode
```

### CODE REVIEW

File: .readthedocs.yml [hunk 1]
```diff
@@ -2,8 +2,13 @@ version: 2
...
     - requirements: requirements.txt
 formats: []
-
+build:
+  os: ubuntu-22.04
+  apt_packages:
+  - graphviz
+  - plantuml
+  tools:
+    python: '3.11'
```
Comment (ikerexxe) [inline .readthedocs.yml:14]: You are missing a line break

**Review by thalman — COMMENTED** [review]
Except for the new line commented by Iker LGTM
Thanks

**Review by thalman — APPROVED** [review]
ACK

**Review by ikerexxe — APPROVED** [review]
LGTM. Thanks for including this plugin!

**Pull request merged**